examples/rcbilling-demo/src/pages/paywall/index.tsx

```diff
@@ -111,14 +111,24 @@ const PaywallPage: React.FC = () => {
 
     // How do we complete the purchase?
     try {
-      const { customerInfo } = await purchases.purchase({
+      const { customerInfo, redemptionInfo } = await purchases.purchase({
         rcPackage: pkg,
         purchaseOption: option,
       });
 
       console.log(`CustomerInfo after purchase: ${customerInfo}`);
+      console.log(
```

Comment:
Do we need this console.log?

Reply:
yup, or lint will fail for property not being used. We can remove it once we use it somewhere/somehow